lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php

```diff
@@ -127,4 +136,30 @@ protected function generateMigration(Configuration $configuration, InputInterfac
 
         return $path;
     }
+
+    protected function manageCustomTemplate(Configuration $configuration, InputInterface $input, OutputInterface $output)
+    {
+        if ($tpl = $input->getOption('template')) {
```

Comment:
Most of doctrine projects are dropping support for PHP < 7.1, we simply forgot to do it on doctrine/migrations. See #537